jans-cedarling/bindings/cedarling_c/tests/test_cedarling.c

```
@@ -667,28 +667,250 @@ void test_trusted_issuer_loading_info() {
+    CedarlingResult auth_result;
+    ret = cedarling_authorize_unsigned_batch(instance_id, valid_batch, &auth_result);
+    TEST_ASSERT(ret == 0, "Batch call executes successfully");
+    if (ret == 0) {
+        TEST_ASSERT(auth_result.data != NULL, "Batch response has data");
+        char* result_str = auth_result.data;
+        printf(" Batch response: %.200s...\n", result_str);
+        TEST_ASSERT(strstr(result_str, "\"batch_id\"") != NULL, "Response carries batch_id");
+        // Each item lands as {"Ok":{decision:true,...}}; expect three envelopes.
+        int ok_count = 0;
+        const char* p = result_str;
+        while ((p = strstr(p, "\"Ok\":{")) != NULL) { ok_count++; p += 6; }
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)